osbuild/main_cli.py

```diff
@@ -191,6 +193,8 @@ def osbuild_cli() -> int:
             else:
                 # if we had monitor.error() we could use that here
                 monitor.log(f"manifest {args.manifest_path} failed\n", origin="osbuild.main_cli")
+            # log overall result
+            monitor.overall_result(fmt.output(manifest, r, object_store))
```

Comment:
I guess the alternative would be to keep doing what we do in `osbuild-composer`  and use the `--montor-fd` and `--json` and read the progress from one fd and the json result from stdout but it feels slightly strange. but it might be a starting point

Reply:
I close this for now in favor of the approach to use `--monitor-fd`